framework/base/Model.php

```diff
@@ -356,6 +356,10 @@ public function validate($attributeNames = null, $clearErrors = true)
             $attributeNames = $this->activeAttributes();
         }
 
+        if(!is_array($attributeNames)) {
```

Comment:
@developeruz maybe it's better to put space between `if` and `(` for PSR-2 stuff?